_includes/js.html

```diff
@@ -0,0 +1,66 @@
+<!-- Essentials -->
```

Comment:
I would break this up a little more, it looks like the essentials are already grouped. just break them off into there own and then we can cut down on all the extra load.